www/source/partials/docs/_using-hab-encryption.html.md.erb

```diff
@@ -1,6 +1,6 @@
 # <a name="using-encryption" id="using-encryption" data-magellan-target="using-encryption">Supervisor and Encryption</a>
 
-By default, a Supervisor will run with no security. It will communicate with other Supervisors in cleartext, and it will allow any user to apply new configuration without authentication. While this is beneficial for quickly illustrating the concepts of Habitat, users will want to run production deployments of Habitat Supervisor networks with more security.
+By default, a Supervisor runs unsecured. It communicates with other Supervisors in plain text, and it allows any user to apply new configuration without authentication. While this is beneficial for demonstrating the concepts of Habitat, users will want to customize the security on production deployments of Habitat Supervisor networks.
```

Comment:
I think "cleartext" is probably more illustrative here as an antonym to "ciphertext". The term "plain text" is less common in a cryptographic context and suffers from confusion with the [plain text format](https://en.wikipedia.org/wiki/Plain_text). At the very least, it should be ["plaintext", with no spaces](https://en.wikipedia.org/wiki/Plaintext).